cli/pkg/up/up.go

```diff
@@ -13,23 +17,43 @@ import (
 	template2 "github.com/rancher/rio/pkg/template"
 )
 
-func readFileInPath(cwd, file string) ([]byte, error) {
+func readFileInPath(cwd, file, filePath string) ([]byte, error) {
```

Comment:
Seems like `cwd` should be either a http:// url or local directory.  I don't think you need a third argument.